tests/main/services-restart/task.yaml

```
@@ -0,0 +1,35 @@
+summary: Check that snap restarts affects only running services
+
+execute: |
+    echo "Installing the service snap"
+
+    # we are using systemd-notify indicate the service is active, this is
+    # currently not allowed by daemon-notify interface, so we may as well just
+    # install in devmode
+    "$TESTSTOOLS"/snaps-state install-local test-snapd-service-restart --devmode
+
+    echo "We can see all services running"
+    for id in 1 2 3 4; do
+        systemctl status snap.test-snapd-service-restart.service$id | MATCH "running"
```

is-active exits with 0 when a service is active, so we can use a simpler version of the check:

```suggestion
        systemctl is-active snap.test-snapd-service-restart.service$id
```